README.md

```diff
@@ -231,6 +232,28 @@ KIP17QueryOptions options = new KIP17QueryOptions();
 options.setSize();
 options.setCursor();
 ```
+### Use KIP-7 API 
+
+You can use KIP-7 API through caver-java-ext-kas. You can send a KIP-7 API request to the KAS as shown below.
+
+```java
+public void getContractList() {
+    try {
+        Kip7ContractListResponse
+        }
```

Comment:
```suggestion
    } catch(ApiException e) {
        //handle error
    }
```
I think it will be great if the example is similar with other snippets in README. 🙂